Uno.UI solution fails to build for WASM in Debug configuration

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

If the matter is security related, please disclose it privately via https://github.com/nventive/Uno/security/
-->

## Current behavior
Building Uno.UI.csproj in the Uno solution in Debug fails with
```
7>UI\Xaml\FrameworkElement.wasm.cs(337,28,337,52): error CS0428: Cannot convert method group 'OnGenericPropertyUpdated' to non-delegate type 'string'. Did you intend to invoke the method?
```
related to this line:
``` csharp
#if DEBUG
			, ChangedCallbackName = OnGenericPropertyUpdated
#endif
```
Cursory investigation indicates there may be other issues as well.

<!-- Describe how the issue manifests. -->

## Expected behavior

<!-- Describe what the desired behavior would be. -->
Solution builds.

## How to reproduce it (as minimally and precisely as possible)

<!-- Please provide a **MINIMAL REPRO PROJECT** and the **STEPS TO REPRODUCE**-->

## Environment

<!-- For bug reports Check one or more of the following options with "x" -->

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

## Anything else we need to know?

<!-- We would love to know of any friction, apart from knowledge, that prevented you from sending in a pull-request -->

--- a/src/Uno.UI/UI/Xaml/FrameworkElement.wasm.cs
+++ b/src/Uno.UI/UI/Xaml/FrameworkElement.wasm.cs
@@ -334,7 +334,7 @@ namespace Windows.UI.Xaml
 		[GeneratedDependencyProperty(
 			Options = FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure
 #if DEBUG
-			, ChangedCallbackName = OnGenericPropertyUpdated
+			, ChangedCallbackName = nameof(OnGenericPropertyUpdated)
 #endif
 		)]
 		public static readonly DependencyProperty MarginProperty = CreateMarginProperty();
@@ -352,7 +352,7 @@ namespace Windows.UI.Xaml
 			DefaultValue = Xaml.HorizontalAlignment.Stretch,
 			Options = FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure
 #if DEBUG
-			, ChangedCallbackName = OnGenericPropertyUpdated
+			, ChangedCallbackName = nameof(OnGenericPropertyUpdated)
 #endif
 		)]
 		public static readonly DependencyProperty HorizontalAlignmentProperty = CreateHorizontalAlignmentProperty();
@@ -369,7 +369,7 @@ namespace Windows.UI.Xaml
 			DefaultValue = Xaml.HorizontalAlignment.Stretch,
 			Options = FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure
 #if DEBUG
-			, ChangedCallbackName = OnGenericPropertyUpdated
+			, ChangedCallbackName = nameof(OnGenericPropertyUpdated)
 #endif
 		)]
 		public static readonly DependencyProperty VerticalAlignmentProperty = CreateVerticalAlignmentProperty();
@@ -386,7 +386,7 @@ namespace Windows.UI.Xaml
 			DefaultValue = double.NaN,
 			Options = FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure
 #if DEBUG
-			, ChangedCallbackName = OnGenericPropertyUpdated
+			, ChangedCallbackName = nameof(OnGenericPropertyUpdated)
 #endif
 		)]
 		public static readonly DependencyProperty WidthProperty = CreateWidthProperty();
@@ -403,7 +403,7 @@ namespace Windows.UI.Xaml
 			DefaultValue = double.NaN,
 			Options = FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure
 #if DEBUG
-			, ChangedCallbackName = OnGenericPropertyUpdated
+			, ChangedCallbackName = nameof(OnGenericPropertyUpdated)
 #endif
 		)]
 		public static readonly DependencyProperty HeightProperty = CreateHeightProperty();
@@ -420,7 +420,7 @@ namespace Windows.UI.Xaml
 			DefaultValue = 0.0d,
 			Options = FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure
 #if DEBUG
-			, ChangedCallbackName = OnGenericPropertyUpdated
+			, ChangedCallbackName = nameof(OnGenericPropertyUpdated)
 #endif
 		)]
 		public static readonly DependencyProperty MinWidthProperty = CreateMinWidthProperty();
@@ -438,7 +438,7 @@ namespace Windows.UI.Xaml
 			DefaultValue = 0.0d,
 			Options = FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure
 #if DEBUG
-			, ChangedCallbackName = OnGenericPropertyUpdated
+			, ChangedCallbackName = nameof(OnGenericPropertyUpdated)
 #endif
 		)]
 		public static readonly DependencyProperty MinHeightProperty = CreateMinHeightProperty();
@@ -455,7 +455,7 @@ namespace Windows.UI.Xaml
 			DefaultValue = double.PositiveInfinity,
 			Options = FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure
 #if DEBUG
-			, ChangedCallbackName = OnGenericPropertyUpdated
+			, ChangedCallbackName = nameof(OnGenericPropertyUpdated)
 #endif
 		)]
 		public static readonly DependencyProperty MaxWidthProperty = CreateMaxWidthProperty();
@@ -473,7 +473,7 @@ namespace Windows.UI.Xaml
 			DefaultValue = double.PositiveInfinity,
 			Options = FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure
 #if DEBUG
-			, ChangedCallbackName = OnGenericPropertyUpdated
+			, ChangedCallbackName = nameof(OnGenericPropertyUpdated)
 #endif
 		)]
 		public static readonly DependencyProperty MaxHeightProperty = CreateMaxHeightProperty();
@@ -485,11 +485,11 @@ namespace Windows.UI.Xaml
 		}
 		#endregion
 
-		private static void OnGenericPropertyUpdated(object dependencyObject, DependencyPropertyChangedEventArgs args)
+		private void OnGenericPropertyUpdated(DependencyPropertyChangedEventArgs args)
 		{
 			if (FeatureConfiguration.UIElement.AssignDOMXamlProperties)
 			{
-				((FrameworkElement)dependencyObject).UpdateDOMProperties();
+				UpdateDOMProperties();
 			}
 		}
 

